HANDLING-REPORTS.md

```diff
@@ -0,0 +1,37 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+Reporting to the OpenJS Foundation either concerns a matter with the Foundation or is considered an escalation where insufficient resolution was provided by a project.
+
+## Channels
+* Email: report@openjsf.org
+  * This email address is for aliasing (like tsc@nodejs.org) to roles in handling reports and escalation.
+  *  We can use groups.io
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email
+* Repo: openjs/moderation (not provisioned)
+  * A form submission could open an issue in the repo, to allow folks to discuss in context
+## Confidentiality and record-keeping
+Everything reported and discussed is confidential unless explicitly stated. For any facts to be made public, the reportee’s explicit permission is needed. All reports should be recorded, together with the discussion of it. A private moderation repo would satisfy the need for recordkeeping.
+## Steps/Escalation
+1. A report is filed: This is the first step, mostly defined in Channels above.
+2. The right people are alerted: See Roles.
+3. Contact person assigned: See Roles.
+4. Report is acknowledged: The contact person responds to let the reportee know that the report is being discussed.
+5. Facts are gathered: Time is allocated to collect information in one place to make sure everyone involved has access.
+6. Facts are discussed: The facts are discussed in context. Also see What to look forward to for guidelines on appropriate response.
+7. An action to be taken is arrived at: The action to be taken is decided by consensus.
```

Comment:
Who can/should participate in this consensus? Everyone in the CPC, along with e.g. observers who are "free to attend meetings and participate in the work of the CPC as well as the consensus seeking process"? Or is this a decision of some smaller committee? And how about CPC members that may be closely affected by the report, are they expected to participate, or to recuse themselves?

Reply:
Punting handling of the consensus process to other sections.

Reply:
@keywordnew I think the discussion in the CPC meeting last week was to move all of this into https://github.com/openjs-foundation/cross-project-council/tree/master/proposals/stage-1/CODE_OF_CONDUCT.  I'm just asking since there is still discussion here.

Reply:
@mhdawson After that discussion, this PR was modified to add this HANDLING-REPORTS.md document to https://github.com/openjs-foundation/cross-project-council/tree/master/proposals/stage-1/CODE_OF_CONDUCT.

Sorry if I misunderstood. How can I correct it?